public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -113,6 +102,25 @@ export class PanelQueryRunner {
     );
   }
 
+  private getTransformationsStream = (withTransforms: boolean): MonoTypeOperatorFunction<PanelData> => {
+    return inputStream =>
```

Comment:
Just out of curiosity: I remember some months ago you have used the standard RxJS naming convention with $ representing a stream (`input$`). Why are we no longer following this pattern?

Reply:
I removed that because I received comments about confusion with Angular :(